params/cluster.go

```diff
@@ -188,6 +194,40 @@ var defaultPushNotificationServers = []string{
 	"5ffc34d5ffda180d94cd3974d9ed2bb082ede68f342babdbe801ceffb7da902087d43f9aa961c7b85029358874c08ef04ecad9f1d95a1f0e448cbdd5d04350c7",
 }
 
+func loadFleetsFromFile(filepath string) (FleetsMap, error) {
+	// Read the JSON file to populate the supportedFleets map
+	file, err := os.Open(filepath)
+	if err != nil {
+		err = pkgerrors.Wrap(err, "failed to open fleets json file")
+		return nil, err
+	}
+
+	defer func() {
+		err = errors.Join(err, file.Close())
+	}()
+
+	var overrideFleets FleetsMap
```

Comment:
Wondering if it's possible to change FleetsMap to []Fleet with an `environment` field in `Fleet`.

Reply:
@kaichaosun the map is here to find a fleet by name, and prevent having multiple fleets with the same name.

And I'm not sure what you mean by `environment` 🤔

Reply:
I was thinking use an array of Fleets for the FleetsMap, so in the FleetInfo, there is a new field `environment`, value could be `status-prod`, `status-staging`, `waku-sandbox`, etc.

Reply:
ok, so `environment` instead of `name`. It is actually a name of the fleet, not just environment.
And I don't think it's needed. As I said, using a map prevents us from having multiple fleets with the same name.